crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -115,6 +118,47 @@ pub struct StandardMaterial {
     pub unlit: bool,
     pub alpha_mode: AlphaMode,
     pub depth_bias: f32,
+
+    /// The height map used for parallax mapping.
+    ///
+    /// Black is the tallest, white deepest.
+    ///
+    /// To improve performance, set your `height_map`'s [`Image::sampler_descriptor`]
+    /// filter mode to `FilterMode::Nearest`, as [this paper] indicates, it improves
+    /// perfs a bit.
+    ///
+    /// [this paper]: https://www.diva-portal.org/smash/get/diva2:831762/FULLTEXT01.pdf
+    #[texture(11)]
+    #[sampler(12)]
+    pub height_map: Option<Handle<Image>>,
+
+    /// How deep the offset introduced by the height map should be.
+    ///
+    /// Default is `0.1`, anything over that value may look distorted.
+    /// Lower values lessen the effect.
+    pub parallax_depth: f32,
+
+    /// Which algorithm to use for the final step of parallax mapping.
+    ///
+    /// We recommend that all objects use the same [`ParallaxAlgorithm`], to avoid
+    /// duplicating and running two shaders.
+    pub parallax_algorithm: ParallaxAlgorithm,
+
+    /// In how many layers to split the height maps for parallax mapping.
+    ///
+    /// If you are seeing jaggy edges, increase this value.
+    /// However, this incures a performance cost.
```

Comment:
TODO: specify by what kind of perf cost. (increasing number of texture lookups, but only in special circumstances.